_posts/0000-01-02-ehmadjawad.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
Talha anjum's mood was f***** by some random boy in last concert in isloo. He said "MArd ka bacha hai tou yahn aa ke bat kr"